webapp-django/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -0,0 +1,163 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is virtually identical except I replaced RAW SQL with the Django model.